### PR TITLE
Respect container element when parsing

### LINKED ISSUE #103: How to render just list items (a set of <li>)?
I had an issue with Interweave I did solve, but I'm wondering if I did it correctly.

### What didn't work
```jsx
ReactDOM.render(<><Markup content="<li>one</li><li>two</li>" tagName="fragment" /><li>three</li><li>four</li></>, container);
```
```html
one two<li>three</li><li>four</li>
```

### What worked
```jsx
ReactDOM.render(<><Markup content="<li>one</li><li>two</li>" allowElements tagName="fragment" /><li>three</li><li>four</li></>, container);
```
```html
<li>one</li><li>two</li><li>three</li><li>four</li>
```

### PR DESCRIPTION
**Pull request by elieux**
This is an attempt to resolve #103. Please comment.

### CODE REVIEW

File: packages/core/src/Interweave.tsx [hunk 1]
```diff
@@ -13,9 +13,10 @@ export default function Interweave(props: InterweaveProps) {
     matchers = [],
     onAfterParse = null,
     onBeforeParse = null,
-    tagName = 'span',
+    wrap = true,
```
Comment (milesj) [inline packages/core/src/Interweave.tsx]: This should be the inverse, `noWrap`, as writing `wrap={false}` is ugly.

File: packages/core/src/Interweave.tsx [hunk 2]
```diff
@@ -13,9 +13,10 @@ export default function Interweave(props: InterweaveProps) {
...
     ...parserProps
   } = props;
+  const tagName = parserProps.tagName || 'div';
```
Comment (milesj) [inline packages/core/src/Interweave.tsx]: Should stay `span` otherwise its a breaking change.

File: packages/core/src/Markup.tsx [hunk 3]
```diff
@@ -24,7 +24,7 @@ export default function Markup(props: MarkupProps) {
     mainContent = emptyContent;
   }
 
-  if (tag === 'fragment') {
+  if (!wrap) {
```
Comment (milesj) [inline packages/core/src/Markup.tsx]: Also breaking. Need to keep the fragment condition.

File: packages/core/src/Interweave.tsx (hunk 2 above)
Comment (elieux) [reply packages/core/src/Interweave.tsx]: Then it should be documented. https://milesj.gitbook.io/interweave/ says under *Markup & Interweave props*:

> tagName (string | 'fragment') - The HTML element tag name to wrap the output with. Defaults to "div".

I will remove this commit.

File: packages/core/src/Markup.tsx (hunk 3 above)
Comment (elieux) [reply packages/core/src/Markup.tsx]: Makes sense. Will fix.

File: packages/core/src/Interweave.tsx (hunk 1 above)
Comment (elieux) [reply packages/core/src/Interweave.tsx]: Yeah, makes sense. Will change.

**Comment by elieux** [comment]
Updated with requested changes.

**Comment by milesj** [comment]
Looks good, just need to fix the failing build.

**Comment by elieux** [comment]
I'll try to get to that sometime this week.

**Comment by milesj** [comment]
I'll take this over the finish line.

**Comment by elieux** [comment]
Yeah, sorry about that and thanks. There were other things IRL that needed my attention.

**Pull request merged**